lib/rex/parser/nokogiri_doc_mixin.rb

```diff
@@ -231,6 +238,10 @@ def end_document
       end
     end
 
+    def save_notes
+      db.report_notes(@notes)
```

Comment:
Let's update this to take an array of plain old ruby hash objects instead of the real Rails objects